src/autowiring/C++11/cpp11.h

```diff
@@ -2,25 +2,17 @@
 #pragma once
 
 // The reason this header exists is due to the asymmetric availability of C++11 on our
-// various compiler targets.  In particular, none of the typical headers for C++11 support
-// exist when building with libstdc++ on Apple, so we have to disable it across the board.
-#ifdef __APPLE__
-  #include <ciso646>  // detect std::lib
-  #ifdef _LIBCPP_VERSION // clang + libc++
-    #define STL11_ALLOWED 1
-  #else // clang + libstdc++
-    #define STL11_ALLOWED 0
-  #endif
-#else // gcc or MSVC
-  #define STL11_ALLOWED 1
-#endif
+// various compiler targets.
+
+#define STL11_ALLOWED 1
```

Comment:
Oh that was done out of caution given existing projects could rely on it. In platform, I removed one use of it in FileSystemUtil.cpp only recently.

Nonetheless, we probably shouldn't have users relying on it. I just checked and neither leapipc nor LeapHTTP use it, so I've gone ahead and removed here.